app/src/pages/authoring/[resourceType]/[id].tsx

```diff
@@ -170,6 +204,28 @@ export default function ResourceAuthoringPage() {
             <ArtifactFieldInput label="name" value={name} setField={setName} />
             <ArtifactFieldInput label="title" value={title} setField={setTitle} />
             <ArtifactFieldInput label="description" value={description} setField={setDescription} />
+            {resourceType === 'Measure' && (
+              <Select
+                label={
+                  <Group spacing="xs">
+                    library
+                    <Tooltip label="Only draft libraries with a valid url may be selected.">
```

Comment:
I LOVE the tooltip! I think it's a good way to inform the user why certain options are disabled and it looks really good. Great work!